test/test-server/ccloud_handlers.go

```diff
@@ -98,14 +98,18 @@ func (c *CloudRouter) HandleMe(t *testing.T, isAuditLogEnabled bool) http.Handle
 			org.Marketplace = &ccloudv1.Marketplace{Partner: ccloudv1.MarketplacePartner_AWS}
 		}
 
+		accounts := environments
+		if os.Getenv("CONFLUENT_CLOUD_EMAIL") == "new@user.com" {
+			accounts = []*ccloudv1.Account{}
```

Comment:
We're trying to move away from this old nomenclature.
```suggestion
			environments = []*ccloudv1.Account{}
```